docs/taxonomy/knowledge/contribution_details.md

```diff
@@ -67,28 +67,32 @@ For knowledge submissions, we need a `qna.yaml` file and an `attribution.txt` fi
 
 For the current version of the taxonomy, version 3, here are the available fields:
 
+!!! note
+    Tokens in the case of context, questions, and answers can fit to "words" but it's specificly tokens are the limitations.
+
 Key | Type | Required | Constraints | Value | Notes
 --|--|--|--|--|--
 `version` | Y | integer | - | `3` | The taxonomy schema version used in the `qna.yaml` file. Defined in [instructlab/schema](https://github.com/instructlab/schema)
 `created_by` | Y | string | - | Your GitHub username | -
 `domain` | Y | string | - | Knowledge sub-category | The knowledge domain which is used in prompts to the teacher model during synthetic data generation. The domain should be brief such as the title to a textbook chapter or section.
 `seed_examples` | Y | array | at least 5 sets | null | This is a collection of questions and answers with context from the knowledge document that InstructLab uses to generate data synthetically.
-`context` | Y | string | < 500 words | A chunk of information from the original knowledge document | This should be a copy-paste from the Markdown version of your document
+`context` | Y | string | < 500 tokens | A chunk of the document showing off the different **unique** content to help guide the teacher model. If you have only text, that's one thing, but if you have tables or other content be sure to add that too. | This should be a copy-paste from the Markdown version of your document
 `questions_and_answers` | Y | array | at least 3 pairs per context | null | This is a collection of questions and answers.
-`question` | Y | string | > 250 words | A question related to the context | Questions are things you'd expect someone to ask the model based on the context given. This will be used for synthetic data generation.
-`answer` | Y | string | > 250 words | An answer for the question | Answers are what you'd like the model to give as an answer. It will not be an exact answer the model always gives.
-`document_outline` | Y | string | - | A brief summary of the document | -
+`question` | Y | string | > 250 tokens | A question related to the grounded in the relivant context | Questions are things you'd expect someone to ask the model based on the context given. This will be used for synthetic data generation.
```

Comment:
```suggestion
`question` | Y | string | > 250 tokens | A question related to the grounded in the relevant context | Questions are things you'd expect someone to ask the model based on the context given. This will be used for synthetic data generation.
```